### DISCUSSION TITLE
How to predict values of ocular drug concentration after oral administration by PK-sim?

### DISCUSSION THREAD

**Discussion by Lillian-hu** [Open-Systems-Pharmacology/Forum#1822] [Questions & Problems]
Hi all
I’m learning to use PK-SIM. And I have a couple of questions. 
I’d like to predict values of ocular drug concentration after oral administration by PK-sim. But I found that the default 'Organ' seems to have no eyes. Is there a place where eyes can be added? Alternatively, PK sim is currently unable to predict eye concentration after oral administration.
Looking forward to your reply very much. Thanks in advance.
<img width="711" alt="捕获" src="https://github.com/user-attachments/assets/43ca1943-cf7c-4f61-8444-6b8158af6335">

Kind regards,
Lillian

**Comment by AndreDlm** [comment]
Dear Lillian,

Currently, there is no compartment for the eyes in PK-Sim. To simulate ocular drug concentrations, there are two workarounds: 
* In PK-Sim, you could replace an existing organ that is irrelevant for your simulation (e.g., gonads) with the eye. Make sure to adjust **all** anatomical and physiological parameters accordingly so that they reflect the eye
* You could export your model to MoBi and add a new organ for the eye. The advantage of doing this in MoBi would be more flexibility. For example, you are not limited to the 4 compartmental substructure of typical organs (blood cells, plasma, interstitial and intracellular) and you could define any number of compartments that are relevant for the eye. You can find further information [here](https://github.com/Open-Systems-Pharmacology/Forum/discussions/1287)

I hope this helps.

Best,
André

**Reply by Lillian-hu** [reply]
Dear André

Thank you for your reply. It's very  helpful.  And I will try to add a new organ for the eye using  MoBi.  😀

Kind regards,
Lillian
